elyra/pipeline/component.py

```diff
@@ -61,49 +63,15 @@ def __init__(self, ref: str, name: str, type: str, value: str, description: str,
         self._ref = ref
         self._name = name
 
-        # Set default value according to type
-        type_lowered = type.lower()
-        if type_lowered in ['str', 'string']:
-            control_id = "StringControl"
-            type = 'string'
-            if not value:
-                value = ''
-        elif type_lowered in ['int', 'integer', 'number']:
-            control_id = "NumberControl"
-            if not value:
-                value = 0
-        elif type_lowered in ['bool', 'boolean']:
-            control_id = "BooleanControl"
+        if control_id == "BooleanControl":
```

Comment:
On looking at this further, I think this block can be removed as it will be covered during the properties JSON template render.